package.json

```diff
@@ -30,10 +30,27 @@
       "title": "CodeBing Configuration",
       "type": "object",
       "properties": {
-        "codebing.searchprovider": {
-          "type": "string",
-          "default": "https://www.bing.com/search?q={query}",
-          "description": "A string to an online search engine. {query} is replaced with the user search."
+        "codebing.searchproviders": {
+          "title": "Search providers",
+          "type": "object",
+          "description": "A string to an online search engine. {query} is replaced with the user search. First word is used as selecter (b or g by default).",
```

Comment:
Should be something like: `A dictionary with search engines, mapping a key to a search url. In the search url {query} is replaced with the user search. Use the key as first word in the search input to specify the search engine.`